docs/geometry.rst

```diff
@@ -265,3 +265,12 @@ single tile.
    .. automethod:: inspect
 
    .. automethod:: compare
+
+Generic Detector
+----------------
+
+Generic detector may represent an unconstrained detector of your choice.
+
+.. autoclass:: GenericDetector
+
+   .. autoattribute:: expected_data_shape
```

Comment:
We should document the `from_simple_description` method, along with the list of methods from the base class which are supported (at least write_crystfel_geom, get_pixel_positions, plot_data_fast, position_modules_fast, output_array_for_position_fast, inspect).

I would guess *compare* also works, but we might want a note that it only makes sense between geometries with the same number & size of tiles.

Reply:
Compare does work indeed! I've checked this already. I can image the use case for *one-module-one-tile detector* as rather usual.

Reply:
I guess I added everything you've mentioned